pkgs/servers/sql/pgbouncer/default.nix

```diff
@@ -10,8 +10,9 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ libevent openssl c-ares ];
+  buildInputs = [ libevent openssl c-ares systemd ];
   enableParallelBuilding = true;
+  configureFlags = [ "--with-systemd" ];
```

Comment:
As this package seems to be cross-platform (`platforms.all`), this should probably be gated behind `stdenv.isLinux`

Reply:
Fixed